ministryofjustice/analytics-platform-control-panel
# Create role oidc provider policy

## What

Added an OIDC provider section to user Role trust policies. This is intended to replace the current SAML provider and trust policy, but the SAML section has been left in place for now to maintain compatibility until the SAML provider is removed.

## How to review

1. Run the API
2. Create a user
3. Verify that the user's role trust policy has an OIDC provider section, with the provider referencing the OIDC provider, and the user's OIDC sub/auth0 ID used as a Condition block.
